info_test.go

```diff
@@ -122,14 +123,14 @@ func TestProgramInfo(t *testing.T) {
 
 			if name == "proc" {
 				_, ok := info.CreatedByUID()
-				qt.Assert(t, ok, qt.IsFalse)
+				qt.Assert(t, qt.IsFalse(ok))
 			} else {
 				uid, ok := info.CreatedByUID()
 				if testutils.IsKernelLessThan(t, "4.15") {
-					qt.Assert(t, ok, qt.IsFalse)
+					qt.Assert(t, qt.IsFalse(ok))
 				} else {
-					qt.Assert(t, ok, qt.IsTrue)
-					qt.Assert(t, uid, qt.Equals, uint32(os.Getuid()))
+					qt.Assert(t, qt.IsTrue(ok))
+					qt.Assert(t, qt.Equals(uid, uint32(os.Getuid())))
```

Comment:
Nit: is the `uint32` conversion still needed?

Reply:
yes, here is the signature of Equal function `Equals[T any](got, want T) Checker`, so got and want should have a same type.